### PR TITLE
Implement and document dependency auto-install script

### PR DESCRIPTION
**Pull request by amada95**
The script automatically detects package manager, reads aero's sysroot dependency list from a file for said package manager, and installs said dependencies from the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,14 +47,18 @@ Aero. If you are using windows, its highly recommended to use WSL 2.
...
-- `qemu` (optional: required if you want to run it in the Qemu emulator)
 - `nasm`
+- `qemu` (optional: required if you want to run it in the Qemu emulator)
 
-If you are building Aero with sysroot then the following dependencies are additionally required:
+If you are building Aero with sysroot, run the following helper script to install additional dependencies.
 ```sh
-# Arch Linux:
-sudo pacman -S bash coreutils make patch tar gzip binutils gcc git subversion mercurial curl wget xz nasm mtools meson perl m4 texinfo groff gettext expat bison flex help2man openssl gperf rsync libxslt python python-pip python-mako xcb-proto xorriso boost-libs cmake
+./tools/deps.sh
```
Comment (Andy-Python-Programmer) [inline README.md:55]: The file is actually present in the `deps` directory. So the path would be `./tools/deps/deps.sh` or just move the file to the `tools` for simplicity.

Comment (amada95) [reply README.md:55]: Alright, I went ahead and fixed this.

**Comment by amada95** [comment]
Requested minor fix was made!

**Comment by amada95** [comment]
Fixed `No such file or directory` error caused by broken relative paths

**Comment by amada95** [comment]
Converted to draft because of known-issues I will work on as soon as I have time:
- packages that fail to install simply show as "done" and are not communicated to user
- `pacman -S` confirmation prompts need to be bypassed

**Comment by amada95** [comment]
This should be ready for review now. So much for not enough time :P

**Comment by amada95** [comment]
Requested changes made, should be good to go

**Review by Andy-Python-Programmer — APPROVED** [review]
LGTM!

**Comment by Andy-Python-Programmer** [comment]
Thanks @amada95!

**Pull request merged**